include/robots/ev3/mindstorms_imu.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+// BoB robotics includes
+#include "net/connection.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// EV3 library
+#include "../../third_party/ev3dev-lang-cpp/ev3dev.h"
+
+// Standard C++ includes
+#include <utility>
+
+namespace BoBRobotics {
+class MindstormsIMU {
+    using degree_t = units::angle::degree_t;
+    using degrees_per_second_t = units::angular_velocity::degrees_per_second_t;
+
+public:
+    MindstormsIMU(const ev3dev::address_type &address = ev3dev::INPUT_AUTO);
+    degree_t getYaw();
```

Comment:
Should these be const? I can imagine that could involve making ``m_IMU`` mutable depending on the const-correctness of EV3Dev

Reply:
Good idea